src/main/kotlin/nl/info/zac/app/zaak/ZaakRestService.kt

```diff
@@ -200,14 +200,19 @@ class ZaakRestService @Inject constructor(
             }
         }
 
-    @PUT
+    @PATCH
     @Path("initiator")
     fun updateInitiator(gegevens: RESTZaakBetrokkeneGegevens): RestZaak {
         val zaak = zrcClientService.readZaak(gegevens.zaakUUID)
         zgwApiService.findInitiatorRoleForZaak(zaak)?.also {
             removeInitiator(zaak, it, ROL_VERWIJDER_REDEN)
         }
-        addInitiator(gegevens.betrokkeneIdentificatieType, gegevens.betrokkeneIdentificatie, zaak)
+        addInitiator(
+            gegevens.betrokkeneIdentificatieType,
```

Comment:
Could consider using named parameters here to increase readability. Although it's pretty clear already I guess..

E.g. 

```
  addInitiator(
            identificationType = gegevens.betrokkeneIdentificatieType,
            [..]
```